src/DynamoCoreWpf/UI/Prompts/FunctionNamePrompt.xaml

```diff
@@ -19,8 +19,13 @@
     <Grid Background="#333">
 
         <StackPanel Margin="15">
-
-            <TextBlock Name ="Indicator" Foreground="WhiteSmoke" Padding="0,15,0,15" FontSize="15" Text="{x:Static p:Resources.CustomNodePropertyWindowTitle}"/>
+            <StackPanel Orientation="Horizontal">
+                <TextBlock Name ="Indicator" Foreground="WhiteSmoke" Padding="0,15,8,10" FontSize="15" Text="{x:Static p:Resources.CustomNodePropertyWindowTitle}"/>
+                <Border BorderThickness="1" Height="35" Width="35" BorderBrush="WhiteSmoke">
+                    <TextBlock Text="CN" FontSize="18" Foreground="WhiteSmoke" HorizontalAlignment="Center" VerticalAlignment="Center"/>
```

Comment:
@Racel should the CN be localized?

Reply:
@mjkkirschner - i don't believe we localize the term "Custom Node". So, this can be left as it is.

Reply:
Localized meaning it would be loaded from an image file?  I asked @Racel about this originally and she said it was okay to just do in the xaml